app/util/sentry/utils.js

```diff
@@ -106,6 +106,22 @@ function rewriteReport(report) {
   return report;
 }
 
+/**
+ * This function excludes events from being logged in the performance portion of the app.
+ * @param {*} event - to be logged
+ * @returns event or null
```

Comment:
[nit] 
```suggestion
 * @returns {(event|null)}
```

https://jsdoc.app/tags-returns
https://jsdoc.app/tags-param see: "Allows one type OR another type (type union)"